src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/integration/visit/GetVisitBookingDetailsTest.kt

```diff
@@ -142,7 +142,7 @@ class GetVisitBookingDetailsTest : IntegrationTestBase() {
   fun `when visit exists search by reference returns the full booking details for that visit`() {
     // Given
     val reference = "aa-bb-cc-dd"
-    val visitors = listOf(createVisitorDto(visitor1, true), createVisitorDto(visitor2, false), createVisitorDto(visitor3, true))
+    val visitors = listOf(createVisitorDto(visitor1, false), createVisitorDto(visitor2, false), createVisitorDto(visitor3, true))
```

Comment:
I've changed each occurrence of this because it looks to be creating **two** visitors set as the main contact whereas there should only ever be one? (Unless I'm missing something!)